### ISSUE TITLE
Promises/A and sequelize

### ISSUE THREAD

**Issue by gavri** [sequelize/sequelize#288]
How do you get sequelize to play well with q.js which follows the Promises/A standard? I'm creating a deferred object, and passing its node-style callback to sequelize's done() function. Is there a better way to do this? And if not, is something that makes this easier planned for the future?

I was hoping it was possible to get sequelize to emit Promises/A promises because that would mean that these promises can be combined with other libraries that follow the same standard. for example, if I want to wait on the success of both a filesystem call and a database call, it's easier to combine them if they are the same kind of promises.
